core-bundle/src/Resources/contao/templates/_new/back_end/module_wildcard.html.twig

```diff
@@ -0,0 +1,12 @@
+{% trans_default_domain 'contao_modules' %}
+
+{% if title %}
+    <h1>{{ title }}</h1>
+{% endif %}
+
+<div class="tl_gray">
+    <span class="upper">### {{ ('FMD.' ~ type ~ '.0')|trans }} ###</span>
+    <br>
+    {% set href = path('contao_backend', {do: 'themes', table: 'tl_module', act: 'edit', id, rt: request_token}) %}
```

Comment:
Why do we compile this in the template instead of passing the URL in the data?

Reply:
I usually try to generate routes in the templates; here is my  reasoning:

- They are only generate when needed.
- You have control over arguments.
- You might want to add additional links anyway and its consistent this way.

In this case both variants are fine, I guess, especially because the BE URLs are very specific. Would you rather move this to the controller, instead?

Reply:
@contao/developers WDYT?

Reply:
For me, compiling the URL in the template is fine for these cases.